assignments/README.md

```diff
@@ -7,9 +7,9 @@ This assignment uses life expectancy in Europe grouped by Country (or other, lik
 ## Installing
 
 1. Clone this repo.
-2. Create a virtual environment with `python -m venv .venv`
-3. Activate the virtual environment with `source .venv/bin/activate` or `.venv\Scripts\activate` on Windows
-4. Install its dependencies on editable mode with `pip install -e .[dev]`.
+2. Create a virtual environment with `python -m venv .venv`. If you are using conda, you can create a virtual environment with `conda create --name foundations`.
```

Comment:
do we want to have an anaconda option? My feeling is that virtual environments should be required...

Reply:
Yeah... Anaconda was screwing things up for one of the mentees; at this point, they didn't have the unit on "virtual environments" yet and they are still too green to fully understand how Anaconda was messing with their settings.

The problem was that even if they had python already installed - and even if you make VSCode use that interpreter instead of Anaconda's, Anaconda just overrides their bash profile and runs its own environments.

It's a mess: if you follow the commands directly, Anaconda just creates a venv inside their venv. Suppose you try to deactivate the Anaconda venv. In that case, you either generate another venv inside their "base" venv or you can't find python anymore since Anaconda overrode your PYTHON_PATH when the terminal started.

Me adding Anaconda's commands was a sign of me giving up solving this properly and just trying to unblock the mentee. The mentoring hour would go way beyond the allotted time otherwise.